### PR TITLE
[DROOLS-7287] Failed to parse comments in RHS

### PR DESCRIPTION
**Pull request by tkobayas**
Fixed to get RHS text as-is.

### CODE REVIEW

**Comment by tkobayas** [comment]
Checking test failure...

File: drools-parser/src/test/resources/org/drools/parser/test_CommentLineNumbersInConsequence.drl [hunk 1]
```diff
@@ -21,12 +21,12 @@ rule simple_rule
       Baz()
   then
       //woot
-      first
+      first;
```
Comment (tkobayas) [inline drools-parser/src/test/resources/org/drools/parser/test_CommentLineNumbersInConsequence.drl:24]: Fixed the test case. Invalid Java syntax in RHS doesn't make sense.

**Review by mariofusco — APPROVED** [review]

File: drools-parser/src/main/java/org/drools/parser/ParserStringUtils.java [hunk 2]
```diff
@@ -38,4 +38,8 @@ public static String getTokenTextPreservingWhitespace(ParserRuleContext ctx, Tok
         // tokenStream is required to get hidden channel token (e.g. whitespace). Unlike getTextPreservingWhitespace, this method reflects Lexer normalizeString
         return tokenStream.getText(ctx.start, ctx.stop);
     }
+
+    public static String trimThen(String rhs) {
+        return rhs.substring("then".length());
```
Comment (gitgabrio) [inline drools-parser/src/main/java/org/drools/parser/ParserStringUtils.java:43]: Hi @tkobayas: my understanding is that this method is specially crafted to remove leading `then` from a string, which could be a good idea; but there is not any check/verification: its correct execution depends completely on calling code.
I would add some check to verify that the string actually starts with `then`: wdyt ?

**Review by gitgabrio — CHANGES_REQUESTED** [review]
Thanks @tkobayas for PR. I would suggest only one slight modification

File: drools-parser/src/main/java/org/drools/parser/ParserStringUtils.java (hunk 2 above)
Comment (tkobayas) [reply drools-parser/src/main/java/org/drools/parser/ParserStringUtils.java]: @gitgabrio Thank you for the suggestion! Added the check.

**Pull request merged**